en/components/grid.md

```diff
@@ -667,6 +805,11 @@ Here is a list of all public methods exposed by **igx-grid**:
 |`sort(name: string, direction, ignorecase)`|Sorts a single column.|
 |`sort(expressions: Array)`|Sorts the grid columns based on the provided array of sorting expressions.|
 |`clearSort(name?: string)`|If `name` is provided, clears the sorting state of the corresponding column, otherwise clears the sorting state of all columns.|
+|`pinColumn(columnName: string)`|Pin a column by its name|
+|`unpinColumn(columnName: string)`|Unpin a column by its name|
+|`enableSummaries(fieldName: string, customSummary?: any)`|Enable summaries for the specified column and apply your `customSummary`. If you do not provide `customSummary` default summary for the column data type will be applied.|
```

Comment:
If you do not provide the `customSummary`, then the default summary for the column data type will be applied.